students/johnpharmd/lesson04/dict_lab.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+# Dictionaries01
+dict1 = {'name': 'Chris', 'city': 'Seattle', 'cake': 'Chocolate'}
+print(dict1)
+
+dict1.pop('cake')
+print(dict1)
+
+dict1['fruit'] = 'Mango'
+print(dict1)
+
+print(dict1.keys())
+print(dict1.values())
+print('"cake" is a key in the dictionary:', 'cake' in dict1)
+print('"Mango" is a value in the dictionary:', 'Mango' in dict1.values())
+print()
+
+# Dictionaries02
+dict2 = dict1.copy()
```

Comment:
be aware that `copy` makes a shallow copy of dict. it won't make a difference in this simple dict, but if your values hold list for example then it will cause issues.